packages/server/src/api/routes/view.ts

```diff
@@ -13,6 +13,11 @@ router
     authorized(permissions.BUILDER),
     viewController.v2.create
   )
+  .put(
+    `/api/v2/views`,
```

Comment:
normally we include the view ID in the URL, can make debugging/looking at requests a bit easier from frontend.